buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -398,11 +403,12 @@ def build_multiple_images(
         path: str = ".",
         file: str = "Dockerfile",
         target: Optional[str] = None,
-        use_threading: bool = True,
+        use_threading: bool = False,
```

Comment:
What are the reasons/implications of this?

Reply:
That was left over from some troubleshooting I did. That is fixed.